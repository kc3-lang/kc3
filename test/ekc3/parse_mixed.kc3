require EKC3
puts(EKC3.parse_str("A<%= x %>B<% y %>C<%= raw z %>D"))
