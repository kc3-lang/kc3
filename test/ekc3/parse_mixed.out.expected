fn (buf) do
  EKC3.raw_str "A"
  EKC3.verbose x
  EKC3.raw_str "B"
  y
  EKC3.raw_str "C"
end
